gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -520,39 +524,27 @@ private[extension] object ColumnarToRowLike {
 }
 // This rule will try to add RowToColumnarExecBase and ColumnarToRowExec
 // to support vanilla columnar operators.
-case class VanillaColumnarPlanOverrides(session: SparkSession) extends Rule[SparkPlan] {
+case class InsertColumnarToColumnarTransitions(session: SparkSession) extends Rule[SparkPlan] {
```

Comment:
It seems the previous rule name is more readable. It is used to be compatible with vanilla columnar related things.

Reply:
Thanks for the explanation. 

Can we temporarily make the name describe what it actually does? If it's now converts from columnar to columnar, I will be inclined to use this new name.

I understand it could do more than columnar transitions but we can discuss on whether to add new logics to this rule or into other independent rules at that time.